library/java/net/openid/appauth/browser/BrowserSelector.java

```diff
@@ -51,9 +50,11 @@
     /**
      * The service we expect to find on a web browser that indicates it supports custom tabs.
      */
+    @SuppressWarnings("StringBufferReplaceableByString")
     @VisibleForTesting
-    static final String ACTION_CUSTOM_TABS_CONNECTION =
-            CustomTabsService.ACTION_CUSTOM_TABS_CONNECTION;
+    static final String ACTION_CUSTOM_TABS_CONNECTION = new StringBuilder()
```

Comment:
This needs a comment explaining the purpose of the code, why `CustomTabsService.ACTION_CUSTOM_TABS_CONNECTION` is not used, a reference to the underlying bug so that it can be reverted in the future when that underlying issue is resolved.

Reply:
added explanation and pushed

Reply:
Explanation looks good! This gives me more confidence that someone will revert the change on day when the underlying bug is addressed :)

Reply:
Hopefully soon!

Reply:
Hi @auval, Is there a rough ETA on when this will be merged?

Reply:
Our legal department is reviewing the CLA before signing it, I've asked them to speed it up. I'll update when I have an ETA.